lib/views/Delta.js

```diff
@@ -1,21 +1,24 @@
 var formatter = require('../utils/formatter'),
   moment = require('moment-timezone'),
-  _ = require('lodash');
+  _ = require('lodash'),
+  View = require('./View'),
+  util = require('util');
 
 require('../utils/date-functions');
 
-function Delta (module) {
-  this.moduleConfig = module.moduleConfig;
-  this.data = module.dataSource.data;
-  this.axes = module.axes;
+function Delta () {
```

Comment:
I don't mind really. I left it like this as I'm pretty sure this will support options soon (like the table) but I can always add that later.